docs/src/content/docs/fr/reference/overrides.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Référence des redéfinitions
+description: Une vue d'ensemble de tous les composants et les props des composants supportés par les redéfinitions de Starlight.
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+Vous pouvez redéfinir les composants intégrés à Starlight en spécifiant des chemins vers des composants de remplacement avec l'option de configuration [`components`](/fr/reference/configuration#components) de Starlight.
+Cette page répertorie tous les composants disponibles qui peuvent être redéfinis et fournit des liens vers leurs implémentations par défaut sur GitHub.
+
+Pour en savoir plus, consultez le [guide des redéfinitions de composants](/fr/guides/overriding-components/).
+
+## Props des composants
+
+Tous les composants peuvent accéder à un objet `Astro.props` standard qui contient des informations concernant la page courante.
+
+Pour typer vos composants personnalisés, importez le type `Props` depuis Starlight :
+
+```astro
+---
+import type { Props } from '@astrojs/starlight/props';
+
+const { hasSidebar } = Astro.props;
+//      ^ type: boolean
+---
+```
+
+Cela vous permettra d'obtenir de l'autocomplétion et un typage lors de l'utilisation de `Astro.props`.
+
+### Props
+
+Starlight passera les props suivantes à vos composants personnalisés.
+
+#### `dir`
+
+**Type:** `'ltr' | 'rtl'`
+
+Le sens d'écriture de la page.
+
+#### `lang`
+
+**Type:** `string`
+
+L’étiquette d’identification BCP-47 pour la langue de la page, par exemple `en`, `zh` ou `pt-BR`.
+
+#### `locale`
+
+**Type:** `string | undefined`
+
+Le chemin de base utilisé pour servir une langue. `undefined` pour les slugs de la locale racine.
```

Comment:
As far as I know, there is no french word for "slug" in this context. Also [not translated](https://developer.mozilla.org/fr/docs/Glossary/Slug) in the french version of MDN.